ui/core.js

```diff
@@ -13,6 +13,8 @@
 //>>description: The core of jQuery UI, required for all interactions and widgets.
 //>>docs: http://api.jqueryui.com/category/ui-core/
 //>>demos: http://jqueryui.com/
+//>>css.structure: ../themes/base/core.css
```

Comment:
Yes it is currently selectmenu basicly duplicates ( incorrectly and inconsistently ) the button css. The button re-write applies button css making them consistent with other buttons / form controls.